src/components/application_manager/src/resumption/resumption_data_processor.cc

```diff
@@ -399,8 +473,11 @@ void ResumptionDataProcessor::DeleteCommands(ApplicationSharedPtr application) {
   LOG4CXX_AUTO_TRACE(logger_);
   const uint32_t app_id = application->app_id();
   ApplicationResumptionStatus& status = resumption_status_[app_id];
-
-  for (auto request : status.successful_requests) {
+  auto requests = status.successful_requests;
```

Comment:
@mked-luxoft please take a look at https://github.com/smartdevicelink/sdl_core/pull/2632#discussion_r220526403

and as I understand this PR contains fix from https://github.com/smartdevicelink/sdl_core/pull/2632 
so 2632 can be closed? If yes, close it please